nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -271,7 +271,7 @@ def get_confounds_file(image_file, flag_full_aroma):
 def get_json(confounds_raw_path):
     """Return json data companion file to the confounds tsv file."""
     # Load JSON file
-    return confounds_raw_path.replace("tsv", "json")
+    return str(confounds_raw_path).replace("tsv", "json")
```

Comment:
get_json is not part of the user facing API so this one as well could return path